### PR TITLE
Adds note that KSPM doesn't support govcloud

### PR DESCRIPTION
**Pull request by benironside**
Small change to add a requirement for the KSPM integration

Twin PR: https://github.com/elastic/staging-serverless-security-docs/pull/302 

Previews: [here](https://security-docs_bk_4942.docs-preview.app.elstc.co/guide/en/security/master/kspm.html) and [here](https://security-docs_bk_4942.docs-preview.app.elstc.co/guide/en/security/master/get-started-with-kspm.html)

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]
Normally, I'd advise against hardcoding URLs into docs since they're subject to change (especially repo URLs), but this one should stay pretty stable, so no issues here! LGTM 🚢🚢🚢

File: docs/cloud-native-security/kspm.asciidoc [hunk 1]
```diff
@@ -13,6 +13,7 @@ This integration supports Amazon EKS and unmanaged Kubernetes clusters. For setu
 [sidebar]
 --
 * The KSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
+* KSPM is supported on commercial cloud only. Government cloud is not supported (https://github.com/elastic/enhancements[request support]).
```
Comment (tinnytintin10) [inline docs/cloud-native-security/kspm.asciidoc:16]: Maybe we can make this line a bit more specific? Something like: Deploying KSPM on EKS clusters in AWS Government Cloud is not supported.

**Review by tinnytintin10 — CHANGES_REQUESTED** [review]
Just one minor suggestion. Besides that, thanks for getting this out quickly, Ben!

File: docs/cloud-native-security/kspm-get-started.asciidoc [hunk 2]
```diff
@@ -7,6 +7,7 @@ This page explains how to configure the Kubernetes Security Posture Management (
 --
 * The KSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
 * The KSPM integration only works in the `Default` Kibana space. Installing the KSPM integration on a different Kibana space will not work.
+* KSPM is supported on commercial cloud only. Government cloud is not supported (https://github.com/elastic/enhancements[request support]).
```
Comment (benironside) [inline docs/cloud-native-security/kspm-get-started.asciidoc:10]:
```suggestion
* KSPM is not supported on EKS clusters in AWS Government Cloud (https://github.com/elastic/enhancements[request support for government cloud]).
```

File: docs/cloud-native-security/kspm.asciidoc (hunk 1 above)
Comment (benironside) [inline docs/cloud-native-security/kspm.asciidoc:16]:
```suggestion
* KSPM is not supported on EKS clusters in AWS Government Cloud (https://github.com/elastic/enhancements[request support for government cloud]).
```

**Review by nastasha-solomon — APPROVED** [review]

**Pull request merged**